internal/apis/acme/v1alpha2/types_issuer.go

```diff
@@ -405,6 +409,80 @@ type ACMEChallengeSolverDNS01 struct {
 	Webhook *ACMEIssuerDNS01ProviderWebhook `json:"webhook,omitempty"`
 }
 
+type ACMEChallengeSolverHTTP01IngressPodSecurityContext struct {
+	// The SELinux context to be applied to all containers.
+	// If unspecified, the container runtime will allocate a random SELinux context for each
+	// container.  May also be set in SecurityContext.  If set in
+	// both SecurityContext and PodSecurityContext, the value specified in SecurityContext
+	// takes precedence for that container.
+	// Note that this field cannot be set when spec.os.name is windows.
+	// +optional
+	SELinuxOptions *corev1.SELinuxOptions `json:"seLinuxOptions,omitempty" protobuf:"bytes,1,opt,name=seLinuxOptions"`
```

Comment:
**question:** Do we marshal to protobuf ever? I might be wrong though

Reply:
This was just copy-pasted from the k8s defs. I think we could remove them - doesn't look like any of the other structs have protobuf annotations, so we wouldn't be able to marshal that way anyway.